service/src/main/java/bio/terra/landingzone/stairway/flight/LandingZoneStepsDefinitionProviderFactory.java

```diff
@@ -1,18 +1,12 @@
 package bio.terra.landingzone.stairway.flight;
 
 public class LandingZoneStepsDefinitionProviderFactory {
-  public static final String CROMWELL_BASE_DEFINITION_STEPS_PROVIDER_TYPE =
-      "CromwellBaseResourcesFactory";
-  public static final String PROTECTED_DATA_DEFINITION_STEPS_PROVIDER_NAME =
-      "ProtectedDataResourcesFactory";
-
   private LandingZoneStepsDefinitionProviderFactory() {}
 
-  public static StepsDefinitionProvider create(String type) {
+  public static StepsDefinitionProvider create(StepsDefinitionFactoryType type) {
     return switch (type) {
       case CROMWELL_BASE_DEFINITION_STEPS_PROVIDER_TYPE -> new CromwellStepsDefinitionProvider();
```

Comment:
I think it is because they are in the same package if you mean why StepsDefinitionFactoryType is not imported.

Reply:
Thanks, I thought it would still have to be imported even though it is in the same package, but I guess I was wrong! When I checked out the branch, my IntelliJ also wasn't automatically linking but perhaps that is because I had not built the code.